src/doctor_cmd.rs

```diff
@@ -0,0 +1,144 @@
+use crate::def_file;
+use std::process;
+use std::os::unix::fs::MetadataExt;
+use dialoguer::Confirm;
+
+pub enum FixMode { Auto, Never, Prompt }
+
+pub fn run(fix_mode: FixMode) {
+    let mut problem_count: u32 = 0;
+    let mut fixed_count: u32 = 0;
+
+    let defs = def_file::load();
+    let mut fixed_defs = defs.clone();
+
+    if defs.is_empty() {
+        problem_count += 1;
+        print_problem(
+            "No commands or command versions are defined. This is normal if \
+            you've just installed alt. You will need to define some commands \
+            & command versions in order to use alt. See: \
+            https://github.com/dotboris/alt#define-command-versions"
+        );
+        println!();
+    }
```

Comment:
This whole check should be done after the cleanup. That way we'll get a warning if we remove the last command